unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarEditorHUD/Scripts/AvatarEditorHUDController.cs

```diff
@@ -520,13 +602,7 @@ public void GoToMarketplace()
             WebInterface.OpenURL("https://docs.decentraland.org/get-a-wallet");
```

Comment:
can we move this to a constant?

Reply:
Yep! Done in https://github.com/decentraland/explorer/pull/2300/commits/f8e457358c9548fb2836f6640f4efac966f39fd0. Thanks!